shared/checks/oval/mcafee_antivirus_definitions_updated.xml

```diff
@@ -0,0 +1,49 @@
+<def-group oval_version="5.10">
+  <definition class="compliance" id="mcafee_antivirus_definitions_updated" version="1">
+    <metadata>
+      <title>Confirm Existence and Permissions of System Log Files</title>
```

Comment:
Title of this check is not related to antiviruses.

Reply:
hahaha... thanks @yuumasato